### ISSUE TITLE
how to relink trajectories from an existing trajectory dataframe

### ISSUE THREAD

**Issue by alexlib** [soft-matter/trackpy#676]
Sometimes I get multiple broken trajectories and would like to run the linking again over the same data, preserving the existing particles and connecting previous trajectories into newer, longer trajectories. 

Thanks for the great software.

**Comment by BptGrm** [comment]
Have you tried tweaking the `memory` and `search_range` parameters of `tp.link` in a way that could produce long enough trajectories ?

**Comment by alexlib** [comment]
Yes of course. Yet there are sometimes cases which are not connected and a second iteration might help. I just cannot figure out the right way to connect the existing trajectories. 
I could do it in manually by renumbering particles probably but I guess there is a better way.
